portal/src/store/modules/stations.ts

```diff
@@ -98,6 +99,7 @@ export class DisplayStation {
         if (!station.updatedAt) throw new Error(`station missing updatedAt`);
         this.updatedAt = new Date(station.updatedAt);
         this.uploadedAt = _.first(station.uploads.filter((u) => u.type == "data").map((u) => new Date(u.time))) || null;
+        this.attributes = station.attributes.attributes // TODO: fix after BE response fix
```

Comment:
This all looks great, but what's this BE response fix in reference to?

Reply:
<img width="810" alt="Screenshot 2022-06-02 at 19 14 29" src="https://user-images.githubusercontent.com/31142342/171676315-9c171ca6-93b9-4cab-9a41-0c007f5ff0f9.png">

Reply:
We have "attributes" prop in 2 levels of indentation.
Do you plan to add anything else besides attributes to the 'attributes' property?

Reply:
Yes, that's intentionally there as a placeholder after having been burned in the past.